.github/workflows/python-app.yml

```diff
@@ -14,31 +14,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.6
+    - name: Set up Python 3.9
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: 3.9
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         pip install flake8 flake8-docstrings
     - name: Lint with flake8
       run: |
-        flake8 . --count --max-complexity=10  --max-line-length=127  \
-        --per-file-ignores="__init__.py:F401,D104 */solver_execution_error/main.py:E999 \
-        */generator_execution_error/main.py:E999 match.py:E221 setup.py:D102,D100 \
-        */problems/*/verifier.py:D102 */problems/*/parser.py:D102 tests/*:D102 tests/__init__.py:D104 \
-        scripts/battle:E501,C901" \
-        --docstring-convention numpy --show-source --statistics --ignore=D105,D401
+        flake8 . 
 
   test:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.6
+    - name: Set up Python 3.9
```

Comment:
Is there a new dependency that requires the use of python in version 3.9 in the CI pipeline? Since [RHEL 8 still provides `python36` as a default version](https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8-3), it would be nice if we do not break downwards compatibility if not necessary.

Reply:
The test and execute parts definitely need 3.9 to run the code, the linting might not though. Locally it seems to work with 3.6 but I'm not entirely sure if older versions of flake8 can lint newer features reliably.